README.md

```diff
@@ -221,7 +220,7 @@ Marker(
 )
 ```
 
-In the example above, when `location` changes the recomposition will be triggered. 
+In the example above, when the `position` attribute in the MarkerState changes, the recomposition will be triggered. 
```

Comment:
How about adding something like this to make it clear:

```
LaunchedEffect(Unit) {
    repeat(10) {
        delay(5.seconds)
        val old = singaporeState.position
        singaporeState.position = LatLng(old.latitude + 1.0, old.longitude + 2.0)
    }
}
```

"In the example above, recomposition occurs as `MarkerState.position` is updated with different values over time", shifting the Marker around the screen.